packages/domain/source/server/todos/todo-list.js

```diff
@@ -1,8 +1,10 @@
 Space.eventSourcing.Aggregate.extend('Todos.TodoList', {
 
-  fields: {
-    name: String,
-    todos: [Todos.TodoItem]
+  fields() {
```

Comment:
Yes, after the #51 is merged. I will maintain this to always work with current APIs. Lets leave this PR open until #51 is merged.